pennylane/registers.py

```diff
@@ -0,0 +1,67 @@
+"""
+This module contains the :func:`registers` function to build the registers for a given dictionary of registers
+"""
+
+from .wires import Wires
+
+
+def registers(register_dict, _start_wire_index=0):
+    """Returns the registers for the given dictionary of registers. The registers are a dictionary
```

Comment:
```suggestion
    """Returns a collection of wire registers when given a dictionary of register names
    and sizes.
    
    The registers are a dictionary
```

Reply:
```suggestion
    """Returns a collection of wire registers when given a dictionary of register names
    and sizes (number of qubits in the register).
    
    The registers are a dictionary
```

@trbromley just a small amendment to your suggestion